apps/dashboard/app/models/files.rb

```diff
@@ -36,6 +48,48 @@ def self.stat(path)
     }
   end
 
+  def self.can_download_as_zip?(path, timeout: Configuration.file_download_dir_timeout, max_download_as_zip_size: Configuration.file_download_dir_max)
```

Comment:
`max_download_as_zip_size` should be renamed, perhaps to just `limit` or `directory_size_limit`

its actually a misnomer to call it a lmit on the zip_size because it is a limit on the size of the directory before compressing